index.js

```diff
@@ -77,19 +77,19 @@ module.exports = function(sass) {
 
     var sassOptions = {
       file: includePathSearcher.findFileSync(this.inputFile, this.inputPaths),
-      includePaths: this.inputPaths,
+      loadPaths: [...this.inputPaths, process.cwd(),  path.join(process.cwd(), 'node_modules')],
```

Comment:
Load paths now need to specify all directories where sass should look for files

Reply:
Would including all of node_modules and all of the root dir be cause for concern in regard to perf?

Reply:
I do not think so and didn't see a perf issue